controller/src/main/java/io/pravega/controller/server/rest/generated/api/ScopesApi.java

```diff
@@ -99,6 +99,23 @@ public Response createStream(@ApiParam(value = "Scope name",required=true) @Path
         return delegate.createStream(scopeName,createStreamRequest,securityContext);
     }
     @DELETE
+    @Path("/{scopeName}/readergroups/{readerGroupName}")
+    
+    
+    @io.swagger.annotations.ApiOperation(value = "", notes = "Delete a reader group", response = void.class, tags={ "ReaderGroups", })
+    @io.swagger.annotations.ApiResponses(value = { 
+        @io.swagger.annotations.ApiResponse(code = 204, message = "Successfully deleted the reader group", response = void.class),
```

Comment:
Ok, it is `204` as the body response is empty, right?

Reply:
Yes, that's correct.